modules/network-observability-viewing-dashboards.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// network_observability/network-observability-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-viewing-dashboards_{context}"]
+= Viewing Network Observability metrics dashboards
+On the *Overview* tab in the {product-title} console, you can view the overall aggregated metrics of the network traffic flow on the cluster. You can choose to display the information by node, namespace, owner, pod, and service. Filters and display options can further refine the metrics.
+
+. In *Observe* -> *Dashboards*, select the *Netobserv* dashboard.
+. View distillations of the network traffic metrics in the following categories, with each having the subset per node, namespace, and source and destination:
```

Comment:
Not sure if there's a reason it's written like this, or if it can lose the second "and"

```suggestion
. View distillations of the network traffic metrics in the following categories, with each having the subset per node, namespace, source, and destination:
```

Reply:
I think in conversation, source and destination get lumped together as hand-in-hand but I agree they don't necessarily need to be written like this.